test/package.json

```diff
@@ -1,12 +1,28 @@
 {
-  "name": "bedrock-test",
+  "name": "bedrock--test",
```

Comment:
This is going to be mistaken for a typo by someone at some point. Can we have a better name?

Reply:
+1 to fixing that somehow.

Reply:
I understand concern.  Suggestions?  This is an edge case just for bedrock itself.  npm didn't like installing `bedrock-test` inside of `bedrock-test`.  `__bedrock_test__`, `bedrock-test_`, `bedrock-t-e-s-t`?  Such a silly problem with silly solutions.

Reply:
Using `bedrock-test-test`.  Good enough for now.